specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/roles.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Is this file original from "2020-02-01-preview"? I don't think we need old SRBAC API any more.